modules/travel_pay/app/controllers/travel_pay/application_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class ApplicationController < ::ApplicationController
+    #include ActionController::Cookies
+    #include ActionController::RequestForgeryProtection
+    service_tag 'travel-pay'
+
+    #protect_from_forgery with: :exception
+
+    #skip_before_action :authorize
+    skip_before_action :authenticate
+    skip_before_action :verify_authenticity_token
```

Comment:
@department-of-veterans-affairs/backend-review-group I've disabled the `:verify_authenticity_token` here since we're using this module as an API only. I just wanted to confirm with y'all that:

If we're using a React frontend and hitting a Rails endpoint, the default authenticity_token is not relevant.

Is that true?

Reply:
@department-of-veterans-affairs/backend-review-group Thoughts?

Reply:
@kjduensing What benefit does skipping this before_action provide? What alternative CSRF protection is there for this?

Reply:
Yes! You will not have an authentication token coming from an external api.

Reply:
I also do not see any user information, I would agree to skip_verification_token.

Reply:
I don't follow that logic - all of vets-api is API-only, with a separate React frontend, and CSRF is still relevant.

Reply:
Thanks all! @patrickvinograd I'd read that somewhere, and also seen some other modules skip the before_action. What you say makes sense. @rmtolmach one of the benefits is that I can use postman to more easily make calls to post/put/delete routes, which I've had trouble with before. 

Sounds like I misunderstood, I'll remove the skip_before_action for the authenticity token. 

Thanks again everyone.

Reply:
@kjduensing I do see it used quite a bit throughout the codebase. What is, or will be, calling TravelPay? vets-website or something else on the backend?

General guidance on CSRF from Identity team (with the caveat that it's often more complicated that this):
- If a frontend calls the route, it needs CSRF protection
- If a backend service calls the route, it may not need CSRF

Reply:
I see it used as well. I was using what I thought was a closely related example as a reference and that example had skipped it. I likely misunderstood their use case when I was referencing it. The fe will call the travel pay backend, so I'll need to "unskip" the verify_authenticity_token action.